<launch>
    <arg name="marker_length" default="0.038" />
    <arg name="aruco_id" default="2" />
    <arg name="matrix_file" default="$(find book_detector)/config/matrix.npy" />
    <arg name="coefficients_file" default="$(find book_detector)/config/coefficients.npy" />

    <node name="MatrixCalibration" pkg="camera_arm_calibration" type="MatrixCalibration.py" output="screen">
        <param name="marker_length" value="$(arg marker_length)" />
        <param name="aruco_id" value="$(arg aruco_id)" />
        <param name="matrix_file" value="$(arg matrix_file)" />
        <param name="coefficients_file" value="$(arg coefficients_file)" />
    </node>
</launch>